### PR TITLE
test: add bootstrap for tests to load essential generis config

### PR DESCRIPTION
**Pull request by gitromba**
There are some essential system constant provided by `generis` and are required in some of the unit tests. By introducing a bootloader for unit tests we ensure these constants are loaded and available upon running these tests.

### CODE REVIEW

**Comment by gabrielfs7** [comment]
Can we move the bootstrap to generis and just adapt phpunit.xml file?

https://github.com/oat-sa/extension-tao-test/pull/351/files#r481148602

**Comment by gitromba** [comment]
[CI being is broken](http://qa.kitchen.it.taocloud.org/blue/organizations/jenkins/ci-pipeline-cg-extension-pr%2Fextension-tao-test/detail/PR-350/3/pipeline/96) is expected in this state. I'll tackle that part once we have a consensus on the solution.

**Comment by gitromba** [comment]
> Can we move the bootstrap to generis and just adapt phpunit.xml file?
> 
> https://github.com/oat-sa/extension-tao-test/pull/351/files#r481148602

Sounds reasonable. I'll make the changes.

**Review by gabrielfs7 — APPROVED** [review]
LGTM. Will test soon. :)

**Review by boajer — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Comment by boajer** [comment]
Shouldn't `manifest` reflect new dependency from `generis` ?

**Review by mike-ionut-mihai-sandu-tao — CHANGES_REQUESTED** [review]
good job, 
but we are missing manifest changes :) 
version bump
dependency version update
also it looks like a good solution

File: manifest.php [hunk 1]
```diff
@@ -31,7 +31,7 @@
     'label' => 'Test core extension',
     'description' => 'TAO Tests extension contains the abstraction of the test-runners, but requires an implementation in order to be able to run tests',
     'license' => 'GPL-2.0',
-    'version' => '14.3.0',
+    'version' => '14.4.0',
     'author' => 'Open Assessment Technologies, CRP Henri Tudor',
     'requires' => [
         'generis' => '>=12.15.0',
```
Comment (mike-ionut-mihai-sandu-tao) [inline manifest.php:37]: i saw the update , great job , we still need something to be finished :
https://github.com/oat-sa/generis/pull/832
updates the version to  : 'version' => '13.3.0',
let's put that value here also :) 
thank you

**Review by mike-ionut-mihai-sandu-tao — CHANGES_REQUESTED** [review]
great job, i last comment

File: manifest.php (hunk 1 above)
Comment (gitromba) [reply manifest.php]: Thanks Mike! I've done this as well.

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]

**Pull request merged**